.storybook/assets/sass/wp-admin.scss

```diff
@@ -0,0 +1,4 @@
+@import 'https://raw.githubusercontent.com/WordPress/WordPress/master/wp-admin/css/common.css';
+@import 'https://raw.githubusercontent.com/WordPress/WordPress/master/wp-admin/css/dashboard.css';
+@import 'https://raw.githubusercontent.com/WordPress/WordPress/master/wp-admin/css/edit.css';
+@import 'https://raw.githubusercontent.com/WordPress/WordPress/master/wp-admin/css/forms.css';
```

Comment:
This certainly feels nicer than installing all of WP via Composer just for VRT, but I don't really like the idea of using the latest styles for every build as it reduces the reproducibility/stability of these tests. Given the very strict nature of the comparisons, I think we should have a fixed version for use with our reference images. We could easily update these URLs to use a specific version instead, but then we would also be handling a dependency version outside of npm/composer. Is there a way we can use a sass variable for the WP version here and set that via config in `package.json`? We would probably need to tweak the webpack config that Storybook uses to provide that, but it sounds possible?

Reply:
I just found this https://github.com/webpack-contrib/sass-loader#additionaldata which looks like it would do the trick.

What do you think about adding something like a `config.storybook_wordpress_version` key to `package.json` and then referencing this as a `process.env.npm_package_config_storybook_wordpress_version` for the above sass variable?

Reply:
Yes, it's a solid plan. I have implemented it.